web/api/py/codechecker_api/setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import setup, find_packages
+from os import path
+
+# io.open is needed for projects that support Python 2.7
+# Python 3 only projects can skip this import
+from io import open
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
```

Comment:
```suggestion
with open('README.md', encoding='utf-8') as f:
```
This way `here` variable can be removed.